integrationtest/mapper/orm/src/test/java/org/hibernate/search/integrationtest/mapper/orm/automaticindexing/association/bytype/AbstractAutomaticIndexingAssociationBaseIT.java

```diff
@@ -207,6 +207,20 @@ public AbstractAutomaticIndexingAssociationBaseIT(IndexedEntityPrimitives<TIndex
 
 	protected abstract boolean isAssociationLazyOnContainingSide();
 
+	protected boolean isFlushCausingAdditionalWork() {
```

Comment:
The name of this method isn't great... we don't know which flush we're talking about, but you mean one specific flush on a specific embedded association. Maybe `isEmbeddedAssociationChangeCausingWork()` would be better?

Regardless, I suspect you don't need to override this method: you could just implement it right here with `!isAssociationOwnedByContainedSide() && !isAssociationMultiValuedOnContainingSide()`...  That's the same implementation as `isElementCollectionAssociationsOnContainingSide`, but I suspect that's just a coincidence.